src/igvfd/tests/data/inserts/biosample.json

```diff
@@ -27,6 +27,7 @@
         "biosample_ontology": "motor neuron",
         "disease_ontology": "motor neuron disease",
         "sex": "male",
-        "date": "2022-02-02"
+        "date": "2022-02-02",
+        "source": "be960dc7-1912-4ff5-8b1a-a7e2e1b2533e"
```

Comment:
Please add biosample that points to a lab instead of a source so that we can test that the linkto works properly?

Reply:
It seems if I use a uuid of a lab it works, but the name of a lab does not. This is likewise true for using a source as source.

Reply:
using "labs/{lab name}" works but for some reason source name is not recognized